README.md

```diff
@@ -12,10 +12,43 @@ npm install content-tag
 
 ## Usage
 
+### Node (CommonJS)
+
 ```js
 let { Preprocessor } = require('content-tag');
 let p = new Preprocessor();
 let output = p.process('<template>Hi</template>');
+
+console.log(output);
+```
+
+### Node (ESM)
+
+wasm-pack (the tool used to build the wasm module), does not support node with ESM, so in node ESM, you still need to use require.
+
+```js
+import { createRequire } from 'node:module';
+
+let require = createRequire(import.meta.url);
```

Comment:
I'm assuming this isn't required any more since your test is importing it directly? https://github.com/embroider-build/content-tag/pull/18/files#diff-bc02260cf90b00b1ff7d9edb08aab803db721f2ccfb25f3d60fbfcd30074b129R8

Reply:
correct, I forgot to update this when I backed out of the conditional `exparts.*` splitting

Reply:
updated!